modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -439,6 +440,18 @@ public void testAddObjectAction() throws Exception {
 				RandomTestUtil.randomString(), name,
 				ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE, false));
 
+		AssertUtils.assertFailure(
+			ObjectActionDescriptionException.class,
+			"Description must be less than 75 characters",
+			() -> _addObjectAction(
+				RandomTestUtil.randomString(76),
+				RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(),
+				ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE,
+				false));
```

Comment:
@Aquiiiles sort alphabetically by exception name. 

This code should be placed before this [line](https://github.com/liferay-bpm/liferay-portal/blob/fb1bfb7e05fa8b34741b01b31b92f8957e1d9024/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java#L390)

1. **ObjectAction**DescriptionException
2. **ObjectAction**ErrorMessageException